### PR TITLE
Feature/#11 add autocomplete for existing resource

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #11

### CODE REVIEW

**Comment by hieptranquoc** [comment]
@hoatle review for me, please

**Comment by hoatle** [comment]
@hieptranquoc please rebase

**Comment by hieptranquoc** [comment]
@hoatle  rebased and checked. review PR again, please!

File: resources/kubectl.rb [hunk 1]
```diff
@@ -68,6 +68,45 @@
...
+      bash 'add it to the bash profile' do
+        code <<-EOH
+            source /etc/profile.d/bash_completion.sh
+            EOH
+        only_if { node['platform_version'].to_f > 7.0 }
+      end
+    end
+
+    # install kubectl autocomplete to current shell
+    bash 'install kubectl autocomplete current shel' do
```
Comment (hoatle) [inline resources/kubectl.rb]: current shell, not sure if we need this?

Comment (hieptranquoc) [reply resources/kubectl.rb]: @hoatle sr, it has been deleted! I just pushed a new one

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**